### DISCUSSION TITLE
Known working systems

### DISCUSSION THREAD

**Discussion by CoolCat467** [CoolCat467/fix-lwjgl#1] [General]
Here I would like to try to keep a list of all systems this program is known to work properly on.

Raspberry Pi OS 64 bit on Raspberry Pi 4

**Comment by Eagle3820** [comment]
I got this to work on a mac with an M chip (M3 in my case), by installing pipx using brew, then changing the installation from using pip to pipx.  Then to get this to work I had to create a shortcut to the fix_lwjgl_wrapper file located in .local/bin in the Minecraft instance folder, then copy the full path of the shortcut to put in the wrapper commands.  There probably is a better way to do this, but I'm not a mac expert so this is the best I could do.  But at least it does work.

**Reply by CoolCat467** [reply]
Thanks so much for posting! For your particular case of making a shortcut to make it work, I think you can do something like adding python to your PATH variable. Just search "add python to PATH macos" and you should find something helpful.
